pkg/util/label.go

```diff
@@ -13,6 +13,20 @@ func GetLabelValue(labels map[string]string, labelKey string) string {
 	return labels[labelKey]
 }
 
+// MergeLabels merges the labels from 'src' to 'dst'.
```

Comment:
It might be better to add a description of the same key override in the comments.

`MergeAnnotation` may have the same needs.